File: app/src/main/java/ru/buylist/data/Pattern.java
package ru.buylist.data;

public class Pattern {
    private long id;
    private String name;

    // конструктор для создания нового шаблона
    public Pattern() {
        id = System.currentTimeMillis();
    }

    // конструктор для существующего шаблона
    public Pattern(long id) {
        this.id = id;
    }

    public long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: app/src/main/java/ru/buylist/data/entity/Pattern.kt
package ru.buylist.data.entity

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "patterns")
data class Pattern(
        @PrimaryKey var id: Long = System.currentTimeMillis(),
        var title: String = "(Без названия...)",
        var items: String = ""
)